boards/nucleo_f429zi/src/main.rs

```diff
@@ -44,7 +48,7 @@ const FAULT_RESPONSE: kernel::process::PanicFaultPolicy = kernel::process::Panic
 /// Dummy buffer that causes the linker to reserve enough space for the stack.
 #[no_mangle]
 #[link_section = ".stack_buffer"]
-pub static mut STACK_MEMORY: [u8; 0x2000] = [0; 0x2000];
+pub static mut STACK_MEMORY: [u8; 0x4000] = [0; 0x4000];
```

Comment:
Note to future self: we'll work on bringing these stack sizes down when merging `tock-ethernet` into master, by fixing the resource (stack) consumption in the TAP ethernet driver.